### PR TITLE
Update Exposing Non-HTTP Services as RESTful APIs page

### PR DESCRIPTION
**Pull request by KalinduGandara**
$subject

### CODE REVIEW

File: en/docs/learn/examples/rest-api-examples/configuring-non-http-endpoints.md [hunk 1]
```diff
@@ -10,18 +10,19 @@ Following is a sample REST API configuration that we can used to implement this
         <resource methods="POST" url-mapping="/"> 
            <inSequence> 
               <property name="REST_URL_POSTFIX" action="remove" scope="axis2"></property> 
-              <send> 
+              <call>
```
Comment (chanikag) [inline en/docs/learn/examples/rest-api-examples/configuring-non-http-endpoints.md:13]: Shouldn't we set out_only true? With this does the request return?

Comment (KalinduGandara) [reply en/docs/learn/examples/rest-api-examples/configuring-non-http-endpoints.md]: fix it

File: en/docs/learn/examples/rest-api-examples/configuring-non-http-endpoints.md [hunk 2]
```diff
@@ -7,20 +7,20 @@ Following is a sample REST API configuration that we can used to implement this
```
Comment (chanikag) [inline en/docs/learn/examples/rest-api-examples/configuring-non-http-endpoints.md:7]: Following is a sample REST API configuration that we can used to implement this scenario. 
->

Following is a sample REST API configuration that we can use to implement this scenario.

File: en/docs/learn/examples/rest-api-examples/configuring-non-http-endpoints.md [hunk 3]
```diff
@@ -32,8 +32,7 @@ Notice that we have specified the `REST_URL_POSTFIX` property with the value set
 
 Create the artifacts:
 
-1. [Set up WSO2 Integration Studio]({{base_path}}/develop/installing-wso2-integration-studio).
-2. [Create an integration project]({{base_path}}/develop/create-integration-project) with an <b>ESB Configs</b> module and an <b>Composite Exporter</b>.
+{!includes/build-and-run.md!}
 3. [Create the rest API]({{base_path}}/develop/creating-artifacts/creating-an-api) with the configurations given above.
```
Comment (chanikag) [inline en/docs/learn/examples/rest-api-examples/configuring-non-http-endpoints.md:36]:
```suggestion
3. [Create the REST API]({{base_path}}/develop/creating-artifacts/creating-an-api) using the configurations given above.
```

**Review by chanikag — APPROVED** [review]

**Pull request merged**